src/sentry/api/endpoints/project_tagkey_details.py

```diff
@@ -59,7 +59,7 @@ def delete(self, request, project, key):
             self.create_audit_entry(
                 request=request,
                 organization=project.organization,
-                target_object=tagkey.id,
+                target_object=getattr(tagkey, 'id', None),
```

Comment:
`target_object` is nullable:
https://github.com/getsentry/sentry/blob/77229ef71e3e9991a61381690c2feb662d934c9a/src/sentry/models/auditlogentry.py#L92

This is also of questionable utility since this record is going to have been deleted at this point, so I'm not sure what good the primary key is here.